spec/config_spec.rb

```diff
@@ -759,4 +759,13 @@
       end
     end
   end
+
+  describe "#to_yaml" do
+    subject { described_class.load(expected_yml) }
+    let(:expected_yml) { File.join(fixtures_dir,'with_params_and_workers.yml') }
```

Comment:
Eh, I'd swap these two lines, so the code references variables defined above it and reads top down.